results2junit

```diff
@@ -164,6 +183,12 @@ class TestCase(object):
         self.time = result['run_time']
         self.set_status(result)
 
+        # *Sob*. Jenkins intercepts URLs ending in '/run', redirecting them
+        # to a non-useful page. This makes it impossible to view failures
+        # in the run test. Workaround: use a different name in junit. Ugh.
+        if self.name == 'run':
+            self.name = 'docker_run'
+
```

Comment:
This will confuse the hell outta anyone that goes looking for ``subtests/docker_cli/docker_run``.  It also won't match with the ``job_report.html`` which could be confusing.  TBH, it may be easier to just rename the test.  We did the same thing in other cases where there were conflicts (i.e. w/ python reserved words).